CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Release 125.0.6 (In development)
 
+### Additions
+
+- Add alert configuration for KubeJobNotCompleted alert
+  (PR[4128](https://github.com/scality/metalk8s/pull/4128))
```

Comment:
I would reword it a bit like "Make `KubeJobNotCompleted` alert time configurable"